stress/common/src/main/java/alluxio/stress/fuse/FUSEBenchTaskResult.java

```diff
@@ -0,0 +1,202 @@
+package alluxio.stress.fuse;
+
+import alluxio.Constants;
+import alluxio.stress.BaseParameters;
+import alluxio.stress.GraphGenerator;
+import alluxio.stress.Summary;
+import alluxio.stress.TaskResult;
+import alluxio.stress.common.SummaryStatistics;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.io.IOException;
+import java.util.*;
+
+public final class FUSEBenchTaskResult implements TaskResult, Summary {
+  private long mRecordStartMs;
+  private Map<Integer, ThreadCountResult> mThreadCountResults;
+  private BaseParameters mBaseParameters;
+  private FUSEBenchParameters mParameters;
+
+  private Map<Integer, Map<String, SummaryStatistics>> mTimeToFirstByte;
+
+  /**
+   * Creates an instance.
+   */
+  public FUSEBenchTaskResult() {
+    // Default constructor required for json deserialization
+    mThreadCountResults = new HashMap<>();
+    mTimeToFirstByte = new HashMap<>();
+  }
+
+  @Override
+  public TaskResult.Aggregator aggregator() {
+    return new Aggregator();
+  }
+
+  @Override
+  public GraphGenerator graphGenerator() {
```

Comment:
move this method under the Aggregater class
since the public method aggregator call the private inner class Aggregator